packages/cloud-cognitive/src/components/ComboButton/ComboButton.stories.js

```diff
@@ -17,19 +16,112 @@ import { ComboButton, ComboButtonItem } from '..';
 
+const modifiers = {
+  'Is Delete': { isDelete: true },
+  'No Delete': { isDelete: false },
+  'Has Divider': { hasDivider: true },
+  'No Divider': { hasDivider: false },
+  'Item Disabled': { itemDisabled: true },
+  'Item Active': { itemDisabled: false },
+};
 export default {
   title: getStoryTitle(ComboButton.displayName),
   component: ComboButton,
+  argTypes: {
+    size: {
+      control: {
+        type: 'radio',
+      },
+      options: ['sm', 'md', 'lg'],
+    },
+    // }
+    modifiers: {
+      control: {
+        type: 'select',
+        labels: Object.keys(modifiers),
+      },
+      options: Object.values(modifiers).map((_k, i) => i),
+      mapping: Object.values(modifiers),
+    },
+  },
   subcomponents: {
     ComboButtonItem,
   },
   parameters: { styles },
 };
 
-export const Default = prepareStory(() => (
-  <ComboButton>
-    <ComboButtonItem>ComboButtonItem 1</ComboButtonItem>
-    <ComboButtonItem renderIcon={CloudApp16}>ComboButtonItem 2</ComboButtonItem>
-    <ComboButtonItem>ComboButtonItem 3</ComboButtonItem>
-  </ComboButton>
-));
+const Template = (args) => {
+  console.log(args);
+  return (
+    <ComboButton disabled={args.buttonDisabled} size={args.size}>
+      <ComboButtonItem>ComboButtonItem 1</ComboButtonItem>
+      <ComboButtonItem renderIcon={args.icon && args.icon}>
+        ComboButtonItem 2
+      </ComboButtonItem>
+      <ComboButtonItem disabled={args?.modifiers?.itemDisabled}>
+        ComboButtonItem 3
+      </ComboButtonItem>
+      <ComboButtonItem hasDivider={args?.modifiers?.hasDivider}>
+        ComboButtonItem 4
+      </ComboButtonItem>
+      <ComboButtonItem isDelete={args?.modifiers?.isDelete}>
+        ComboButtonItem 5
+      </ComboButtonItem>
+    </ComboButton>
+  );
+};
+
+// const DividerTemplate = (args) => (
+//   <ComboButton>
+//     <ComboButtonItem>ComboButtonItem 1</ComboButtonItem>
+//     <ComboButtonItem renderIcon={args.icon && args.icon}>
+//       ComboButtonItem 2
+//     </ComboButtonItem>
+//     <ComboButtonItem disabled={args.disabled}>
+//       ComboButtonItem 3
+//     </ComboButtonItem>
+//     <ComboButtonItem>ComboButtonItem 4</ComboButtonItem>
+//     <ComboButtonItemDivider />
+//     <ComboButtonItem danger={args.danger}>ComboButtonItem 5</ComboButtonItem>
+//   </ComboButton>
+// );
+
+// const DisabledTemplate = (args) => (
+//   <ComboButton disabled={args.disabled}>
+//     <ComboButtonItem>ComboButtonItem 1</ComboButtonItem>
+//     <ComboButtonItem renderIcon={args.icon && args.icon}>
+//       ComboButtonItem 2
+//     </ComboButtonItem>
+//     <ComboButtonItem>ComboButtonItem 3</ComboButtonItem>
+//     <ComboButtonItem>ComboButtonItem 4</ComboButtonItem>
+//     <ComboButtonItemDivider />
+//     <ComboButtonItem danger={args.danger}>ComboButtonItem 5</ComboButtonItem>
+//   </ComboButton>
+// );
+
+export const Default = prepareStory(Template, {
+  args: {
+    //hasDivider: false,
+    //isDelete: false,
+    buttonDisabled: false,
```

Comment:
We should use consistent naming and just use `disabled` here. There’s not really any reason I can see for both so let replace instances of `buttonDisabled` with `disabled`. I believe their intention is the same.

![image](https://user-images.githubusercontent.com/1717398/211717994-672035aa-2555-467f-832a-b923e712b1fb.png)

Reply:
Also make sure to add `children: 0` here if you update with the suggested changes. — `children: 0` just means that we are using the first of the options we set.